app/controllers/application_controller.rb

```diff
@@ -9,9 +9,15 @@ class ApplicationController < ActionController::Base
   rescue_from ActiveRecord::RecordNotFound, with: :record_not_found
 
   before_action :authenticate_user!
+  around_action :switch_locale
 
   private
 
+  def switch_locale(&action)
+    locale = params[:locale] || I18n.default_locale
+    I18n.with_locale(locale, &action)
+  end
```

Comment:
тебе нужно поменять значение локали

```suggestion
  def switch_locale(&action)
    locale = params[:locale] || I18n.default_locale
    I18n.locale = locale
  end
```